### ISSUE TITLE
WorkflowInstanceCreationRecord uses different naming for workflow instance key

### ISSUE THREAD

**Issue by Zelldon** [zeebe-io/zeebe#2263]
https://github.com/zeebe-io/zeebe/blob/develop/protocol-impl/src/main/java/io/zeebe/protocol/impl/record/value/workflowinstance/WorkflowInstanceCreationRecord.java#L33

**Comment by Zelldon** [comment]
Might be that this is caused by the naming in the java client API ?

```java

  /**
   * Command to create/start a new instance of a workflow.
   *
   * <pre>
   * zeebeClient
   *  .newCreateInstanceCommand()
   *  .bpmnProcessId("my-process")
   *  .latestVersion()
   *  .variables(json)
   *  .send();
   * </pre>
   *
   * @return a builder for the command
   */
  CreateWorkflowInstanceCommandStep1 newCreateInstanceCommand();

```

**Comment by saig0** [comment]
We should use the same names to reduce confusion. Also, we should rename `key` to `workflowKey`.

**Comment by menski** [comment]
this was fixed with #2657
